strongbox-web-core/src/test/java/org/carlspring/strongbox/rest/ArtifactControllerTest.java

```diff
@@ -0,0 +1,761 @@
+
+        final String md5Local = mdos.getMessageDigestAsHexadecimalString(EncryptionAlgorithmsEnum.MD5.getAlgorithm());
+        final String sha1Local = mdos.getMessageDigestAsHexadecimalString(EncryptionAlgorithmsEnum.SHA1.getAlgorithm());
+
+        System.out.println("MD5   [Remote]: " + md5Remote);
+        System.out.println("MD5   [Local ]: " + md5Local);
+
+        System.out.println("SHA-1 [Remote]: " + sha1Remote);
+        System.out.println("SHA-1 [Local ]: " + sha1Local);
+
+        File artifact = new File("target/partial-foo-3.1.jar");
+        if (artifact.exists())
+        {
+            artifact.delete();
+            artifact.createNewFile();
+        }
+        FileOutputStream output = new FileOutputStream(artifact);
+        output.write(baos.toByteArray());
+        output.close();
+
+        assertEquals("Glued partial fetches did not match MD5 checksum!", md5Remote, md5Local);
+        assertEquals("Glued partial fetches did not match SHA-1 checksum!", sha1Remote, sha1Local);
+    }
+
+    private InputStream getArtifactAsStream(String path,
+                                            String url)
+    {
+        return getArtifactAsStream(path, url, -1);
+    }
+
+    private InputStream getArtifactAsStream(String path,
+                                            String url,
+                                            int offset)
+    {
+        return new ByteArrayInputStream(getArtifactAsByteArray(path, url, offset));
+
+    }
+
+    private byte[] getArtifactAsByteArray(String path,
+                                          String url,
+                                          int offset)
+    {
+        MockMvcRequestSpecification o = given().contentType(MediaType.TEXT_PLAIN_VALUE);
+        int statusCode = 200;
+        if (offset != -1)
+        {
+            o = o.header("Range", "bytes=" + offset + "-");
+            statusCode = 206;
+        }
+
+        MockMvcResponse response = o.param("path", path)
+                                    .when()
+                                    .get(url);
+        Headers allHeaders = response.getHeaders();
+        System.out.println("HTTP GET " + url);
+        System.out.println("Response headers:");
+        allHeaders.forEach(header ->
+                           {
+                               System.out.println("\t" + header.getName() + " = " + header.getValue());
+                           });
+
+        response.then().statusCode(statusCode);
+        byte[] result = response.getMockHttpServletResponse().getContentAsByteArray();
+        System.out.println("Received " + result.length + " bytes.");
+
+        return result;
+    }
+
+    public boolean pathExists(String path)
+    {
+        String url = getContextBaseUrl() + (path.startsWith("/") ? path : '/' + path);
+
+        logger.debug("Path to artifact: " + url);
+
+        //   WebTarget resource = getClientInstance().target(url);
```

Comment:
If such commented out original code is no longer necessary, could you please remove it?